modules/lvgl/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ zephyr_include_directories(${LVGL_DIR}/src/)
 zephyr_include_directories(include)
 
 zephyr_compile_definitions(LV_CONF_INCLUDE_SIMPLE=1)
+zephyr_compile_definitions(_POSIX_C_SOURCE=200809L)
```

Comment:
can we use zephyr_library_compile_definitions ? (so we limit it to this library)?

Reply:
Sure, good call